helpers/loadENV.js

```diff
@@ -1,3 +1,6 @@
-window.location.hostname !== 'dashboard.realdevsquad.com'
-  ? (window.API_BASE_URL = 'https://staging-api.realdevsquad.com')
-  : (window.API_BASE_URL = 'https://api.realdevsquad.com');
+window.API_BASE_URL =
+  window.location.hostname !== 'dashboard.realdevsquad.com'
+    ? window.location.hostname !== 'localhost'
+      ? 'https://staging-api.realdevsquad.com'
+      : 'https://api.realdevsquad.com'
+    : 'https://api.realdevsquad.com';
```

Comment:
Can we please avoid using nested ternary operations?

Reply:
Thanks, fixed